lib/client.js

```diff
@@ -980,7 +980,8 @@ Request.prototype.end = function(fn){
   if ('GET' != this.method && 'HEAD' != this.method && 'string' != typeof data && !isHost(data)) {
     // serialize stuff
     var contentType = this.getHeader('Content-Type');
-    var serialize = request.serialize[contentType ? contentType.split(';')[0] : ''];
+    var contentParser = request.serialize[contentType ? contentType.split(';')[0] : ''];
```

Comment:
Is that a real issue? Sorry, I was just trying to keep within the 80 character line width